Jellyfin.Data/DayOfWeekHelper.cs

```diff
@@ -1,67 +1,21 @@
 #pragma warning disable CS1591
 
 using System;
-using System.Collections.Generic;
 using Jellyfin.Data.Enums;
 
 namespace Jellyfin.Data
 {
     public static class DayOfWeekHelper
     {
-        public static List<DayOfWeek> GetDaysOfWeek(DynamicDayOfWeek day)
+        public static DayOfWeek[] GetDaysOfWeek(DynamicDayOfWeek day)
         {
-            var days = new List<DayOfWeek>(7);
-
-            if (day == DynamicDayOfWeek.Sunday
-                || day == DynamicDayOfWeek.Weekend
-                || day == DynamicDayOfWeek.Everyday)
-            {
-                days.Add(DayOfWeek.Sunday);
-            }
-
-            if (day == DynamicDayOfWeek.Monday
-                || day == DynamicDayOfWeek.Weekday
-                || day == DynamicDayOfWeek.Everyday)
-            {
-                days.Add(DayOfWeek.Monday);
-            }
-
-            if (day == DynamicDayOfWeek.Tuesday
-                || day == DynamicDayOfWeek.Weekday
-                || day == DynamicDayOfWeek.Everyday)
-            {
-                days.Add(DayOfWeek.Tuesday);
-            }
-
-            if (day == DynamicDayOfWeek.Wednesday
-                || day == DynamicDayOfWeek.Weekday
-                || day == DynamicDayOfWeek.Everyday)
+            return day switch
             {
-                days.Add(DayOfWeek.Wednesday);
-            }
-
-            if (day == DynamicDayOfWeek.Thursday
-                || day == DynamicDayOfWeek.Weekday
-                || day == DynamicDayOfWeek.Everyday)
-            {
-                days.Add(DayOfWeek.Thursday);
-            }
-
-            if (day == DynamicDayOfWeek.Friday
-                || day == DynamicDayOfWeek.Weekday
-                || day == DynamicDayOfWeek.Everyday)
-            {
-                days.Add(DayOfWeek.Friday);
-            }
-
-            if (day == DynamicDayOfWeek.Saturday
-                || day == DynamicDayOfWeek.Weekend
-                || day == DynamicDayOfWeek.Everyday)
-            {
-                days.Add(DayOfWeek.Saturday);
-            }
-
-            return days;
+                DynamicDayOfWeek.Everyday => new[] { DayOfWeek.Sunday, DayOfWeek.Monday, DayOfWeek.Tuesday, DayOfWeek.Wednesday, DayOfWeek.Thursday, DayOfWeek.Friday, DayOfWeek.Saturday, DayOfWeek.Sunday },
```

Comment:
was Sunday placed in this statement twice on purpose? (once before Monday and once on the end after Saturday)

Reply:
I see, and yeah I noticed this in some other context but failed to connect the dots. Thank you for the clarification :wink: